README.md

```diff
@@ -41,7 +41,7 @@ Additionally, the library offers an interface to encode SVG images.
 ## Prerequisites
 
 ```
-sudo apt install gcc-c++ cmake librsvg2-dev libx264-dev libgtk-3-dev
+sudo apt install cpp cmake librsvg2-dev libx264-dev libgtk-3-dev
```

Comment:
Shouldn't it be `g++`? `gcc-c++` is Fedora package, Ubuntu's is `g++`.  cpp is preprocessor:

```
apt-cache search cpp | grep cpp
cpp - GNU C preprocessor (cpp)
```

Reply:
> Shouldn't it be `g++`?

You are right, g++ is C++ pack! Sorry. Please check another commits...